static/src/javascripts/projects/commercial/modules/dfp/create-slot.js

```diff
@@ -15,6 +15,17 @@ const inlineDefinition = {
     },
 };
 
+const adSlotToBlockthroughUids = {
+    inline1: '5a98587091-157',
+    inline2: '5a98587869-157',
+    inline3: '5a98587f21-157',
+    inline4: '5a985885d5-157',
+    inline5: '5a98588c9c-157',
+    inline6: '5a985892ed-157',
+    inline7: '5a985899d4-157',
+    inline8: '5a9858a114-157',
```

Comment:
I think there's no limit to the number of inline slots.  Could this be done by a prefix comparison instead?

Reply:
The limit is on Blockthrough' side. we Have to provide them with our slots.

Reply:
I didn't notice that they were all different!